ceph.spec.in

```diff
@@ -248,6 +248,12 @@ on commodity hardware and delivers object, block and file system storage.
 Summary:       Ceph Base Package
 Group:         System Environment/Base
 Requires:      ceph-common = %{epoch}:%{version}-%{release}
+# redhat-lsb-core/lsb-release for lsb_release executable
+%if 0{?suse_version}
+Requires:      lsb-release
+%else
+Requires:      redhat-lsb-core
```

Comment:
I see now that the ticket has more explanation: http://tracker.ceph.com/issues/14906

I'd rather that we dropped the `lsb_release_parse()` function. If we really need the information, there are more modern ways to get it (eg. `/etc/os-release`) that don't depend on `lsb-release`.